### ISSUE TITLE
WoltModalSheet does not follow finger movement smoothly when dismissing

### ISSUE THREAD

**Issue by LiangLuDev** [woltapp/wolt_modal_sheet#269]
When using WoltModalSheet.show, the modal does not follow the finger movement smoothly when attempting to close it by pulling down (similar to ModalBottomSheet). Instead, it requires significant force to pull down and close. I couldn’t find a setting to modify this behavior and am unsure if it can be adjusted.

Additionally, the distance from the top of the modal to the status bar cannot be modified. I have set the forceMaxHeight property, but it does not change the distance.

Could you please provide guidance on how to resolve these issues?

Reproduction Steps:

	1.	Use WoltModalSheet.show to display a modal.
	2.	Attempt to close the modal by pulling it down with a finger.
	3.	Observe the resistance to movement and the inability to smoothly follow the finger.

Expected Behavior:

The modal should smoothly follow the finger movement when pulling down to close, similar to the behavior of ModalBottomSheet. Additionally, the distance from the top of the modal to the status bar should be adjustable.

Actual Behavior:

The modal requires significant force to pull down and close, and the distance to the status bar cannot be modified even when setting the forceMaxHeight property.

Environment:

	•	Library version: 0.7.1
	•	Platform: Mobile
	•	Flutter version: 3.19

Thank you for your assistance!

**Comment by ulusoyca** [comment]
Hello @LiangLuDev Can you check the Custom Modal Type section in ReadMe, and create a custom modal type with a custom `minFlingVelocity `?
https://pub.dev/packages/wolt_modal_sheet#defining-custom-modal-types

**Comment by LiangLuDev** [comment]
> Hello @LiangLuDev Can you check the Custom Modal Type section in ReadMe, and create a custom modal type with a custom `minFlingVelocity `?

Yes, I found this parameter and tried to set this property. Unfortunately, it did not take effect. I also tried it in the test demo you provided, but it did not work either.

**Comment by ulusoyca** [comment]
Yikes! Thanks for reporting. I will take this. Maybe @durannumit can also take a look?

**Comment by durannumit** [comment]
> Yikes! Thanks for reporting. I will take this. Maybe @durannumit can also take a look?

sure I will check according to comments

**Comment by vishna** [comment]
@durannumit @ulusoyca  observing this issue too, I thought it's maybe because I was on old version, but went from 0.7.1 -> 0.11.0, and the issue is still there. Super slow drag, weird resistance effect. I'm quite sure it wasn't there always so maybe it has something to do with Flutter changes 🤔

**Comment by MrLepage** [comment]
This is barely usable in the current state unfortunately

**Comment by andrewwahid** [comment]
Any progress on this issue ?
